pandas/core/generic.py

```diff
@@ -2900,13 +2918,14 @@ def to_sql(
         >>> from sqlalchemy.types import Integer
         >>> df.to_sql('integers', con=engine, index=False,
         ...           dtype={"A": Integer()})
+        3
 
         >>> engine.execute("SELECT * FROM integers").fetchall()
         [(1,), (None,), (2,)]
-        """
+        """  # noqa: 501
```

Comment:
don't you mean E501?

Reply:
Thanks. Maybe this just treats it all `noqa` for all codes.